src/arrayMethodReduce.js

```diff
@@ -3,9 +3,22 @@
 /**
  * Implement method Reduce
  */
+
 function applyCustomReduce() {
   [].__proto__.reduce2 = function(callback, initialValue) {
-    // write code here
+    let prev = initialValue;
```

Comment:
I know that we use `prev` in video, but better rename to `previousValue` or `accumulator` as said in [method](https://developer.mozilla.org/ru/docs/Web/JavaScript/Reference/Global_Objects/Array/Reduce)